### DISCUSSION TITLE
Extension availability

### DISCUSSION THREAD

**Discussion by leesus** [supabase/supabase#4340] [Questions] [answered]
I have 2 projects, both in the same AWS region. One has had the `PG_HASHIDS` extension made available recently, which I'd like to enable so that I can move away from a plv8 solution to accomplish the same thing. Oddly though, my other database doesn't have `PG_HASHIDS` in the list of extensions, so I can't enable it.

I've tried restarting the server which had no effect - any ideas?

**Comment by GaryAustin1** [comment, marked as answer]
pg_hashids was added to the recent Postgres 14 image released.  If your other server is more than a week old it is probably 13.3 (or older).  Right now Supabase does not have an automatic upgrade path with PostGres changes.  If you search here or the discord channel there are discussions on migrating to a new server on your own.  Otherwise maybe support has an option for adding an extension to a specific server, but I don't recall that being discussed anywhere.

**Reply by leesus** [reply]
Thanks. Bizarrely, both projects are from early in the year, so it's a bit strange that one is up to date and one isn't!

**Reply by GaryAustin1** [reply]
Very odd.  Can you see what version they are running?  SELECT version();  
edit:  I just checked my 4 month old project and pg_hashids is not there.

**Reply by leesus** [reply]
Oh, ok, I've got it. The one on PG 14.1 was paused due to inactivity, it must have been upgraded once I restored it. Hopefully the other server will be paused and get an upgrade too.

**Reply by GaryAustin1** [reply]
Wait! So if we let a server pause it will get upgraded!  I'm taking a vacation shortly and won't touch my dev server.... ;)

**Reply by leesus** [reply]
It's the only thing I can think of between the 2 servers.

**Reply by soedirgo** [reply]
We've updated the instructions for migrating to a new project: https://supabase.com/docs/guides/database#migrating-between-projects. That should let you install the latest extensions.
